config/deploy_development.env

```diff
@@ -6,8 +6,8 @@ CARD_CREATOR_PASSWORD=AQECAHjqALewp8JBJNxIQvR4oY795dyG7INaGR1glMsTEgetggAAAJ4wgZ
 
 #      Encrypted Values for API v0.2 and v0.3
 # ---- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----
-ILS_CLIENT_KEY=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAHoweAYJKoZIhvcNAQcGoGswaQIBADBkBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDIvO/ZW0cKkEsbPZJgIBEIA3kh8+qS55SMHXouGPYHa4VbgVbV+lvhhtx5Ej6+b2eDbsj0kqeF+6BT6hOML5fyq8HsOS6UF7Bg==
-ILS_CLIENT_SECRET=AQECAHh7ea2tyZ6phZgT4B9BDKwguhlFtRC6hgt+7HbmeFsrsgAAAGswaQYJKoZIhvcNAQcGoFwwWgIBADBVBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDH42Uqe+Qlmf9kKtVgIBEIAo/TNJI/LDt+Y7mUvxuNeCcYi50lV41ng1MFOyPuAjgwW8k8+oHtMdwQ==
+ILS_CLIENT_KEY=AQECAHjqALewp8JBJNxIQvR4oY795dyG7INaGR1glMsTEgetggAAAHoweAYJKoZIhvcNAQcGoGswaQIBADBkBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDPqlRaJIvps4Tp0USAIBEIA38wxXb3VmxMcgFVakZkb2McezDL6hq8c/gaWLYo+4x9iJHCE8QY4wuOKBufaxurLMd+Fwr7FcjA==
+ILS_CLIENT_SECRET=AQECAHjqALewp8JBJNxIQvR4oY795dyG7INaGR1glMsTEgetggAAAGgwZgYJKoZIhvcNAQcGoFkwVwIBADBSBgkqhkiG9w0BBwEwHgYJYIZIAWUDBAEuMBEEDHxVxF63bLpp3wIiLwIBEIAleC8XwfOC0qq+0sRZMYytWwH2iJ8h/s+qELB7eJK+dHKycVInuA==
```

Comment:
This key/secret does not work for me when I test.

Reply:
Check to see if you encrypted the values correctly.

Reply:
nypl-sandbox keys I used  for encrypted

Reply:
I did encrypted looks keys are fine